packages/wdio-cli/tests/commands/repl.test.js

```diff
@@ -28,22 +27,17 @@ describe('Command: repl', () => {
     })
 
     it('should set the correct browser', async () => {
-        await handler({ browserName: 'foobar' })
+        await handler({ option: 'foobar' })
```

Comment:
Could we test that the handler works properly when the flags are passed?

example:
```js
await handler({ option: 'foobar', platformVersion: 'test', deviceName: 'foo', uuid: '123test' })
```

Reply:
Well, I've added such test for `getCapabilities()` function, so not sure that we really need it here also.